### PR TITLE
Feature/login form design

### PR DESCRIPTION
**Pull request by maaarsas**
Added messages, spinners and other styles to login form to make it nicer

### CODE REVIEW

**Comment by maaarsas** [comment]
image

**Comment by maaarsas** [comment]
Changed login form to be reactive

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -10,34 +11,45 @@ import {ActivatedRoute, Router} from '@angular/router';
...
-
-    this.authService.login(this.username, this.password)
+  onLoginSubmit(): void {
+    if (!this.loginForm.valid) {
+      return;
+    }
+    // indicate loading so the form can be disabled and loading animations shown
+    this.loading = true;
+    // using AuthService send a login request with user email and password
+    this.authService.login(this.loginForm.get('email').value, this.loginForm.get('password').value)
```
Comment (gpiliponyte) [inline src/app/login/login.component.ts]: this is ok, but for larger forms it's better to simply use loginForm.value and pass the object to the service like that

Comment (gpiliponyte) [reply src/app/login/login.component.ts]: it would be {email: "smth", password: "admin"}

**Pull request merged**